randovania/games/fusion/logic_database/Sector 2 TRO.txt

```diff
@@ -464,14 +464,14 @@ Extra - room_id: [14]
   * L0 Hatch to Central Shaft/Door to Dessgeega Dormitory
   * Extra - door_idx: (33,)
   > Door to Collapsed Shaft
-      Trivial
+      Screw Attack or Combat (Intermediate) or Normal Damage ≥ 199
 
 > Door to Collapsed Shaft; Heals? False
   * Layers: default
   * L0 Hatch to Collapsed Shaft/Door to Dessgeega Dormitory
   * Extra - door_idx: (34, 100)
   > Door to Central Shaft
-      Trivial
+      Screw Attack or Combat (Intermediate) or Normal Damage ≥ 199
 
```

Comment:
Does it make sense to use `Can Kill Tough Beak-Weak Enemy` instead?

Reply:
added the template, but still kept the normal damage threshold as an option